Amethyst/Managers/ScreenManager.swift

```diff
@@ -124,6 +124,21 @@ final class ScreenManager<Window: WindowType>: NSObject {
             }
 
             self?.onReflowCompletion?()
+            // If this reflow was caused by a window swap, re-focus the window
+            // so that mouse follows focus is respected if it is set
+            switch event {
```

Comment:
I have to think about this more, but I would like to keep window focusing isolated to the relevant transition coordinator.